packages/plugin/src/rules/require-import-fragment.ts

```diff
@@ -100,11 +126,11 @@ export const rule: GraphQLESLintRule = {
     }
 
     return {
-      FragmentSpread(fragmentSpreadNode) {
-        fragmentSpreadNameNodes.add(fragmentSpreadNode.name);
+      'FragmentSpread .name'(node: GraphQLESTreeNode<NameNode>) {
+        fragmentSpreadNameNodes.add(node);
       },
-      FragmentDefinition(fragmentDefinitionNode) {
-        knownFragmentNames.add(fragmentDefinitionNode.name.value);
+      'FragmentDefinition .name'(node: GraphQLESTreeNode<NameNode>) {
```

Comment:
Why `.name` and not just `Name`?

Reply:
it is because `FragmentDefinition Name` will match

```js
// some abstract GraphQL AST Node
const node = {
  type: 'FragmentDefinition',
  name: { // ✅ matched 1
    type: 'Name'
    // ...
  },
  foo: { // ✅ matched 2
    type: 'Name',
    bar: { // ✅ matched 3
      type: 'Name' 
      // ...
    }
    // ...
  }
  // ...
}
```
2. `FragmentDefinition .name` will match only `node.name`
3. `FragmentDefinition > Name` will match only `node.name` and `node.foo` but not `not.foo.bar`

So for more safety, it should be `FragmentDefinition > .name`, I remember I got some issue due too `wide` selector, so it's better to be explicit while selecting nodes